README.md

```diff
@@ -7,9 +7,15 @@ This app is designed to help maintainers understand a contributors history with
 ## Getting Started
 
 This application requires a GitHub Personal Access Token to be set in the .env file.
-Directions on generating a PAT are available in the [GitHub Documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic) 
+Directions on generating a PAT are available in the [GitHub Documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic)
 
-First, run the development server:
+First you have to install the dependencies:
+
+```bash
+npm install
```

Comment:
We have a lock file, so I think `npm ci` would be better and it them doesn't update the lock file and we have to ask people to remove it from their PR

```suggestion
npm ci
```